src/compat.h

```diff
@@ -10,16 +10,6 @@
 #include <config/bitcoin-config.h>
 #endif
 
-#include <type_traits>
```

Comment:
Should include in bench? https://en.cppreference.com/w/cpp/types/is_default_constructible